src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,7 @@
-// Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { TodoInfo } from '../TodoInfo';
```

Comment:
You should consider adding PropTypes to your components to ensure that you are receiving the correct data and data types.